### PR TITLE
New: Notes - Promotion to Core users [ED-5523]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/regions/panel/pages/menu/views/group.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = Marionette.CompositeView.extend( {
 		const callback = childView.model.get( 'callback' );
 
 		if ( _.isFunction( callback ) ) {
-			callback.call( childView );
+			callback.call( childView, childView );
```
Comment (Nevoss) [inline assets/dev/js/editor/regions/panel/pages/menu/views/group.js:20]: Maybe you don't need the second argument and use the `this` inside the callback?

File: assets/dev/js/editor/regions/panel/pages/menu/menu.js [hunk 2]
```diff
@@ -73,6 +73,22 @@ PanelMenu.addAdminMenu = () => {
...
+			elementor.promotion.showDialog( {
+				headerMessage: __( 'Notes', 'elementor' ),
+				message: __( 'With Notes, teamwork gets even better. Stay in sync with comments, feedback & more on your website.', 'elementor' ),
+				top: '-3',
+				inlineStart: '+10',
+				element: view.$el,
+				actionURL: 'https://go.elementor.com/go-pro-notes/',
+			} );
+		},
+	}, 'navigate_from_page', 'view-page' );
```
Comment (Nevoss) [inline assets/dev/js/editor/regions/panel/pages/menu/menu.js:90]:
```suggestion
	PanelMenu.addItem( {
		name: 'notes',
		icon: 'eicon-commenting-o',
		title: __( 'Notes', 'elementor' ),
		callback: function ( view ) {
			elementor.promotion.showDialog( {
				headerMessage: __( 'Notes', 'elementor' ),
				message: __( 'With Notes, teamwork gets even better. Stay in sync with comments, feedback & more on your website.', 'elementor' ),
				top: '-3',
				inlineStart: '+10',
				element: this.$el,
				actionURL: 'https://go.elementor.com/go-pro-notes/',
			} );
		},
	}, 'navigate_from_page', 'view-page' );
```

File: assets/dev/js/editor/regions/panel/pages/menu/menu.js [hunk 3]
```diff
@@ -73,6 +73,22 @@ PanelMenu.addAdminMenu = () => {
 		],
 	}, { at: 0 } );
 
+	PanelMenu.addItem( {
```
Comment (Nevoss) [inline assets/dev/js/editor/regions/panel/pages/menu/menu.js:76]: @matipojo do you think this promotion should be on module or something?

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/regions/panel/pages/menu/menu.js (hunk 3 above)
Comment (matipojo) [reply assets/dev/js/editor/regions/panel/pages/menu/menu.js:76]: It's OK

**Pull request merged**